sopt/src/test/scala/dagr/sopt/cmdline/ClpReflectiveBuilderTest.scala

```diff
@@ -112,4 +115,28 @@ class ClpReflectiveBuilderTest extends UnitSpec {
       an[IllegalStateException] should be thrownBy _.toCommandLineString
     }
   }
+
+  it should "print v if the argument type was an Option and the value was Some(v)" in {
+    val t = new ClpReflectiveBuilder(classOf[DefaultWithSomeOption])
+    t.argumentLookup.view.foreach {
+      _.toCommandLineString shouldBe "--w 4"
+    }
+  }
+
+  it should "print the special empty or none token for a None value or empty collections" in {
+    // None
+    {
+      val t = new ClpReflectiveBuilder(classOf[DefaultWithOption])
+      t.argumentLookup.view.foreach {
+        _.toCommandLineString shouldBe s"--w ${ReflectionUtil.SpecialEmptyOrNoneToken}"
+      }
+    }
+    // Empty Collection
+    {
```

Comment:
Why not just make these separate tests?  I think it would take up less space than how you've laid them out ;)